src/iosched/fcfs.c

```diff
@@ -110,13 +110,18 @@ int fcfs_destroy(void *iosched_handle)
  */
 int fcfs_open(const char *path, bool open_write, struct dentry **dentry, void *iosched_handle)
 {
+	int ret;
 	struct fcfs_data *priv = (struct fcfs_data *) iosched_handle;
 
 	CHECK_ARG_NULL(path, -LTFS_NULL_ARG);
 	CHECK_ARG_NULL(dentry, -LTFS_NULL_ARG);
 	CHECK_ARG_NULL(iosched_handle, -LTFS_NULL_ARG);
 
-	return ltfs_fsraw_open(path, open_write, dentry, priv->vol);
+	ret = ltfs_fsraw_open(path, open_write, dentry, priv->vol);
+	if (!ret)
```

Comment:
At least in the I/O schedulers we're used to write `if (ret == 0)` to check if a function call succeeded. I think it's good idea to use the same coding pattern, as `! ret` is used to check if an operation failed.

Reply:
Oops, `if (!ret)` is used in libltfs. So I think it is good to use. I'll use `if (ret == 0)`.

I think Brian prefers to use `if (!ret)`, good call, and `if (ret < 0)`, bad call`.

Reply:
Ah, I didn't remember we used that pattern in libltfs. No worries then -- this is a minor issue

Reply:
Fixed in my branch.